recipe/meta.yaml

```
@@ -284,6 +284,42 @@ outputs:
         - osgeo._osr
       commands:
         - python run_test.py
+        # Check .py GDAL utilities
+        # --help not implemented as documented for gdalattachpct.py, gdal_merge.py, gdal_retile.py, gdal_sieve.py, gdalmove.py, gdal_edit.py, gdal_pansharpen.py, gdalcompare.py, ogrmerge.py?
+        # also, --version does not work with e.g. rgb2pct.py...
+        # In addition, some scripts are checked only for existence (gdalattachpct.py, gdal_retile.py, gdal_proximity.py) as they unexpectedly return error codes.
+        - rgb2pct.py --help
+        - pct2rgb.py --help
+        # Neither --help nor --version seem to work
+        # - if not exist %PREFIX%\\Library\\bin\\gdalattachpct.py exit 1  # [win]
```

Maybe try this next:
```suggestion
        # - if not exist %LIBRARY_BIN%\\gdalattachpct.py exit 1  # [win]
```

---

But I believe that's the same.